src/content/en/fundamentals/engage-and-retain/push-notifications/permission-ux.md

```diff
@@ -0,0 +1,177 @@
+permission, so lets take a brief aside to look at both good and bad UX.
+
+## Common Patterns
+
+There have been a few common patterns emerging that should guide / help you when
+deciding what is best for your users and use case.
+
+### Value Proposition
+
+Encouraging a user to subscribe to push messaging at a time where the benefit
+is obvious given the current context is a fantastic way to get users to grant
+permission.
+
+For example, a user has just bought an item on an online store and finished the
+checkout flow, the site can then offer updates on the delivery status.
+
+There are a range of situations where this pattern / approach works:
+- A particular item is out of stock, would you like to notified when it's next available?
+- This breaking news story will be regularly updated, would you like to be notified as the
+story develops?
+- You're the highest bidder, would you like to be notified if you are outbid?
+
+These are all points where the user has invested in your service and there
+is a clear value proposition for them to enable push notifications.
+
+[Owen Campbell-Moore](https://twitter.com/owencm) created a mock site for an airline that
+demonstrates this approach.
+
+After the user has booked a flight asks if the user would like notifications in case of flight
+delays.
+
+![Owen Campbell-Moore's example of good UX for push.](./images/ux-examples/owen/owen-good-example.png){: .device-image .center-image }
+
+Note that this isn't the browser UI as well, this is a custom UI in the web site asking the
+user, allowing the site full control over the messaging to the user.
+
+Another nice touch to Owen's demo is that if the user does click to enable
+notification, the site adds a semi-transparent overlay over the entire page when
+it shows the permission prompt. This helps draw the users attention to the
+permission prompt.
+
+![Owen Campbell-Moore's example of good UX for the permission prompt.](./images/ux-examples/owen/owen-permission-prompt.png){: .device-image .center-image }
+
+The alternative to this example, the **bad UX** for asking permission, is to  request
+permission as soon as a user lands on the airline's site.
+
+![Owen Campbell-Moore's example of bad UX for push.](./images/ux-examples/owen/owen-bad-ux.png){: .device-image .center-image }
+
+No context as to what the notifications are for, if the user wanted to get a
+task done (i.e. check a flight or book a flight), this permission prompt gets in
+the way of that and secondly it's not a good look having a pop-up over the site.
+
+### Double Permission
+
+You may feel that your site has a clear need for push messaging and as a result
+want to ask the user for permission as soon as possible. An example of this
+sites that fit into this category are messaging sites or email clients. You
+receiving a message or email, show a notification.
+
+In these cases, it's worth considering the double permission UX.
+
```

Comment:
I assume you mean the prompt is just an ordinary part of the page and not a blocking pop-up or something. The text is less than clear about that. An image would really help. The link to slack is only good if you're signed in.